controller/base.php

```diff
@@ -69,5 +70,17 @@ public function __construct(helper $helper, ideas $ideas, linkhelper $link_helpe
 		$this->php_ext = $php_ext;
 
 		$this->user->add_lang_ext('phpbb/ideas', 'common');
+
+		// Don't let using Ideas if it's not been properly configured
```

Comment:
This is not needed in the base controller. It should be at the start of each controller classes primary method